Gemfile.lock

```diff
@@ -138,14 +138,14 @@ GEM
     websocket-driver (0.7.0)
       websocket-extensions (>= 0.1.0)
     websocket-extensions (0.1.3)
-    zendesk_apps_support (4.10.0)
+    zendesk_apps_support (4.11.0)
       erubis
       i18n
       image_size
       jshintrb (~> 0.3.0)
       json
-      loofah
-      nokogiri (~> 1.6.8)
+      loofah (~> 2.2.1)
+      nokogiri (~> 1.8.1)
```

Comment:
Both of these had security vulnerabilities. 

![](https://i.imgur.com/oCkv72I.png)